### PR TITLE
Profile page

### PR DESCRIPTION
**Pull request by tobthecreator**

### CODE REVIEW

File: components/InfoCards/DescriptionCard.vue [hunk 1]
```diff
@@ -4,8 +4,8 @@
     :headerString="headerString"
     :subheaderString="subheaderString"
     :subheaderURL="subheaderURL"
-    subheaderOn
-    headerOn>
+    :subheaderOn="subheaderOn"
+    :headerOn="headerOn">
```
Comment (aleckhoury) [inline components/InfoCards/DescriptionCard.vue:8]: I also do this in some areas of my code, not worth dealing with now, but I feel like there's a shorter way in Vue to pass variables through when they have the same name as the propName. No changes needed here, just noting

Comment (tobthecreator) [reply components/InfoCards/DescriptionCard.vue:8]: I made a note to myself to go back and have those also turn off or on if we don't pass any content to the header/subheader

File: pages/dev/u/_username.vue [hunk 2]
```diff
@@ -1,24 +1,89 @@
 <template lang="html">
-  <div class="screen">
+  <div class="screen background-tint">
```
Comment (aleckhoury) [inline pages/dev/u/_username.vue:2]: does this still need to be included if we change the main.css background selector from 'body' to 'html'?

**Review by aleckhoury — APPROVED** [review]

File: pages/dev/u/_username.vue (hunk 2 above)
Comment (tobthecreator) [reply pages/dev/u/_username.vue:2]: almost all of those background tints are left over from when I didn't port most of the page styling to global main.css.  Not needed, but not hurting right now. I'll get them in my next branch

**Pull request merged**